### PR TITLE
Move "settings" Cog to a more visible and understandable location in the menu (issue #993)

### PR DESCRIPTION
**Pull request by JeanneSon**
some changes per the instructions of the issue #993
Unfortunately, I was not able to run E2E tests nor unit tests

### CODE REVIEW

**Comment by JeanneSon** [comment]
## Illustration
#### If the user is in non-project contexts, the cog icon is not displayed. (previously, it was grayed out)
grafik

#### If in a project context, the icon looks like the Share button.
grafik

#### On small devices (like mobile):
grafik

#### On very small devices (like iPhone 5):
grafik

**Review by megahirt — COMMENTED** [review]
The screenshots look good!  And you've made it work with minimal code changes, well done!  We will need to see the E2E and Unit Tests passing in order for this merge in.  I will push your branch to sillsdev/web-languageforge and then the CI will kick off.  This is an unusual situation (PR from personal repo) that our CI doesn't handle, and so we'll have to manually fix it up.

File: src/Site/views/languageforge/theme/default/sass/_global.scss [hunk 1]
```diff
@@ -275,8 +264,41 @@
     }
   }
 
+  #settings-dropdown-button-original {
+    display: none;
+  }
```
[the remark this answers is not in the document]
Comment (JeanneSon) [reply src/Site/views/languageforge/theme/default/sass/_global.scss]: Yes, this is now deleted.

**Review by megahirt — APPROVED** [review]
Looks good!  Approved pending unit and e2e tests passing.

**Comment by megahirt** [comment]
In a conversation with @JeanneSon today, we agreed to created a separate PR #1062 for the purposes of running tests on TeamCity CI.  We can now see that all the tests pass for this PR, and so we can merge this even though the tests won't run for this "outside" PR.

**Pull request merged**